Human.java

```diff
@@ -0,0 +1,25 @@
+public class Human {
+    static int legs = 2;
```

Comment:
I think this is ok. In the future, though, we might consider the scenario of someone having 1 leg or none.